frontend/src/stub/router.js

```diff
@@ -0,0 +1,75 @@
+const express = require('express');
+const reponseRequeteApiSivParticulier200 = require('./fixtures/reponseRequeteApiSivParticulier200');
```

Comment:
J'ai essayer en appelant les fixtures de cypress de cette facon et ca fonctionne, le stub ne doit pas comprendre les alias

```suggestion
const reponseRequeteApiSivParticulier200 = require('../../cypress/fixtures/api/reponseRequeteApiSivParticulier200.json');